bundles/core/pom.xml

```diff
@@ -89,6 +89,10 @@
                             javax.annotation;version=0.0.0,
                             *
                         </Import-Package>
+                        <Embed-Dependency>
+                            commons-text;inline=false;scope=compile|runtime
```

Comment:
Instead of embedding here I suggest we add as a dependency of ui.apps package and install commons-text directly. The jar is already an OSGI bundle.

Reply:
Done, the bundle is now embedded in `ui.apps`.